Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -478,6 +478,9 @@ public float OdometerM
         public float ThrottleIntervention = -1;
         public float DynamicBrakeIntervention = -1;
 
+        // Has dynamic brake movement been paused to give user a chance to react?
+        private bool DynamicBrakePause = false;
```

Comment:
I wonder whether this is really needed. With this we get two notches to switch from braking to throttle in smooth combined controls. I would only expect a pause in throttle-0 and brake-0 (so two keypresses to switch from full throttle to full braking and vice-versa).

Reply:
Well, this behavior is there to mitigate frustration and make the dynamic brakes behave the same as they did in versions from a few days ago. In both cases, you need to hold D until dynamic brakes reach 0, hit D again to disable the dynamic brakes, then hit D a second time to go into the throttle range. The first hit to D to disable the dynamic brakes is vital in making sure nobody accidentally un-sets-up the dynamic brakes and I prefer the second hit as well to make sure I don't start throttling on accident.

Furthermore, I realized I failed to replicate a similar behavior when moving into the dynamic braking range-on those older builds, a double press is required to engage dynamic brakes as well (one press to start set-up, a second press to actually increase the dynamic brake value) but now it's just a single press.

Maybe this can be disabled if there's no set-up notch (with a check for the number of notches on the dynamic brake controller) and/or disabled if the set-up time is small or 0 (no reason to worry about un-setting the dynamic brake if it can be restored immediately). But such a thing is another deviation from the previous standard that players could notice.